gcp_variant_transforms/pipeline_common.py

```diff
@@ -154,6 +156,21 @@ def _get_file_names(input_file):
     return contents
 
 
+def create_file_path_to_file_hash_map(input_patterns, project):
+  # type: (List[str], str) -> Dict[str, str]
+  """Returns a mapping from file path to its MD5 hash."""
+  matches = filesystems.FileSystems.match(input_patterns)
+  all_files = [metadata.path for match in matches
+               for metadata in match.metadata_list]
+  file_path_to_file_hash = {}
+  for input_file in all_files:
+    bucket_name, blob_name = gcsio.parse_gcs_path(input_file)
+    bucket = storage.Client(project).get_bucket(bucket_name)
```

Comment:
Also, as discussed online, I attempted to run this logic with my service account on 'gcp-variant-transforms-test' on the following files:
- gs://gcp-variant-transforms-testfiles/small_tests/valid-4.2.vcf
- gs://gcp-variant-transforms-testfiles/small_tests/valid-4.0.vcf 

The bucket's data is public so I can access the files, but storage.Client(project) command fails on me, presumably because my service account is not associated with the project.

This is probably going to be an occurring scenario for our customers, as I would presume they would want to use public datasets, without being directly involved with the project hosting them (e.g. 1k genomes).

Reply:
Thanks so much for helping testing it! Yes, the `get_bucket` would require bucket level permissions, while the user might only have the object-level permission. Fixed.